spec/features/visitor/visitor_can_adjust_quantity_of_cart_items_spec.rb

```diff
@@ -13,19 +13,75 @@
       expect(page).to have_field("Quantity", with: "1")
     end
 
-    expect(page).to have_content("Item Subtotal: $#{item.price}")
-    expect(page).to have_content("Total: #{item.price}")
+    expect(page).to have_content("Item Subtotal: $#{item.formatted_price}")
+    expect(page).to have_content("Total: $#{item.formatted_price}")
 
     fill_in "Quantity", with: "2"
     click_on "Update"
 
     expect(current_path).to eq(cart_path)
-    save_and_open_page
```

Comment:
remove the debugger

Reply:
Yeah Randy. I think that's a great idea.